docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -630,18 +630,18 @@ To process the received command on the server side, follow these steps:
    void
    playback_action_cmd_received_cb(const char* client_name, const char *request_id, mc_playback_action_e action, void *user_data)
    {
-       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request Id: %s, Playback action: %d\n", client_name, request_id, action);
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request ID: %s, Playback action: %d\n", client_name, request_id, action);
    }
    
    ret = mc_server_set_playback_action_cmd_received_cb(g_mc_server, playback_action_cmd_received_cb, NULL);
    ```
 
-   - To register a callback for playback state change commands, use the `mc_server_set_playback_action_cmd_received_cb()`.
-   - To register a callback for playback position change commands, use the `mc_server_set_playback_position_cmd_received_cb()`.
-   - To register a callback for shuffle mode change commands, use the `mc_server_set_shuffle_mode_cmd_received_cb()`.
-   - To register a callback for repeat mode change commands, use the `mc_server_set_repeat_mode_cmd_received_cb()`.
-   - To register a callback for played item, playback state and playback position change commands in playlist, use the `mc_server_set_playlist_cmd_received_cb()`.
-   - To register a callback for a custom command, use the `mc_server_set_custom_cmd_received_cb()`.
+   - To register a callback for playback state change commands, use `mc_server_set_playback_action_cmd_received_cb()`.
+   - To register a callback for playback position change commands, use `mc_server_set_playback_position_cmd_received_cb()`.
+   - To register a callback for shuffle mode change commands, use `mc_server_set_shuffle_mode_cmd_received_cb()`.
+   - To register a callback for repeat mode change commands, use `mc_server_set_repeat_mode_cmd_received_cb()`.
+   - To register a callback for played item, playback state and playback position change commands in playlist, use  `mc_server_set_playlist_cmd_received_cb()`.
```

Comment:
@haejeongkim 
Kindly add "," before and  
To register a callback for played item, playback state, and playback position change commands in playlist, use  `mc_server_set_playlist_cmd_received_cb()`.

Reply:
Done